src/internal/shoelace-element.ts

```diff
@@ -1,5 +1,6 @@
 import { LitElement } from 'lit';
 import { property } from 'lit/decorators.js';
+import { ScopedElementsMixin } from '@open-wc/scoped-elements';
```

Comment:
open-wc's mixin is gonna break with Lit's SSR if you're headed there next. Lit's mixin is a) smaller, and b) less complex, and c) works more seamlessly with `@lit-labs/nextjs` (at least in my experiments thus far).

I'd consider using [Lit's ScopedRegistryHost](https://github.com/lit/lit/tree/main/packages/labs/scoped-registry-mixin) mixin as your default maybe?

Reply:
And this is why I asked. Did not realize Lit had their own! Especially weird since Open-WC's is also lit specific 🤔

Reply:
scoping always breaks ssr (as there is no way yet to set a scoped registry declaratively - as far as I know)

depends on what you need... lits version always requires the polyfill and does not offer a way to dynamically load/register elements...

but yeah it would be nice of there would be only on scopemixin in the ecosystem but yeah mistakes where made and now we have 2... 🙈

Reply:
3 😞 I made my own that could be used independent of Lit and using bare HTMLElement

If Lit's ScopedRegistryHost is the better default, I'm happy to do that. I don't know where we stand on SSR currently, but it is a question that would need to be answered.